applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_flux_FIC_condition.cpp

```diff
@@ -201,13 +201,14 @@ template <unsigned int TDim, unsigned int TNumNodes>
 void UPwNormalFluxFICCondition<TDim, TNumNodes>::CalculateAndAddBoundaryMassMatrix(
     MatrixType& rLeftHandSideMatrix, NormalFluxVariables& rVariables, NormalFluxFICVariables& rFICVariables)
 {
-    noalias(rFICVariables.PPMatrix) = GeoTransportEquationUtilities::CalculateCompressibilityMatrix(
-        rVariables.Np, rFICVariables.BiotModulusInverse, rVariables.IntegrationCoefficient);
+    BoundedMatrix<double, TNumNodes, TNumNodes> compressibility_matrix =
```

Comment:
I would prefer to use `const auto` here, rather than this long type name:
```suggestion
    const auto compressibility_matrix =
```

I would suggest similar changes in other places, too.

Reply:
changed everywhere except line 1269 in U_Pw_small_strain_element.cpp